web/src/components/navbar/NavBarVerticalLine.tsx

```diff
@@ -0,0 +1,14 @@
+import type { ReactNode } from "react";
+
+export const NavBarVerticalLine = (): ReactNode => (
+  <div
+    className="height-4"
+    style={{
+      width: "2px",
+      background:
+        "linear-gradient(0, rgba(46,65,83,0) 0%, rgba(46,65,83,0) 25%, rgba(46,65,83,1) 25%, rgba(46,65,83,1) 75%, rgba(46,65,83,0) 75%, rgba(46,65,83,0) 100%)",
+    }}
+  >
+    &nbsp;
+  </div>
```

Comment:
Is there a reason we didn't just add this to a pseudo class? It feels like overkill to create a component for this.

Reply:
@seidior had created this component and I just kept it. When you say psuedo class do you mean just make it styling within a the navbar component?

Reply:
Yeah, like just add a `::after` to the CSS rule for the logo image or something.

Reply:
Nah it's not valid for ::after since it needs to be a conditional render, if I recall correctly. And I'm totally okay having this be a component as it improves code readability.

OTOH, there's probably more to do here to clean this up, like moving more of this into USWDS that can be moved in.

Reply:
I don't think I understand why `::after` wouldn't work. We could still manage this with CSS rules, but even still adding this to a rule applying to the logo component would suffice. This is only relevant any time the logo is on the page. The conditional rendering for the logo would be enough.

I'm not super torn, I just think in this instance there's a lot of component boiler plate being added here for what is essentially a CSS change.